lib/client/rpcClient.ts

```diff
@@ -0,0 +1,46 @@
+import { Program, Provider, utils } from "@coral-xyz/anchor";
+import { AutocratProgram, Dao, DaoState, ProgramVersion } from "../types";
+import { PublicKey } from "@solana/web3.js";
+import { FutarchyClient } from "./client";
+import { enrichTokenMetadata } from "../tokens";
+
+export class FutarchyRPClient implements FutarchyClient {
+  private programVersion: ProgramVersion;
+  private rpcProvider: Provider;
+  private constructor(programVersion: ProgramVersion, rpcProvider: Provider) {
```

Comment:
I discovered an issue today which I haddn't thought of, but it's how we know which DAO people are looking to use. This likely means we'll maintain a list (until metadata program) so.....

Reply:
a list of daos with their metadata? Or like you were saying I think in our discussion, maybe we can use the baseToken logo in the interim...

Reply:
Well we don't need the logo as much as we need to know from the list of daos, which one they select to load that data for the UI. Since we fetch them all, the key is just to make sure we're passing it through so we're referencing the data display from their selection.

The logo's I'm not too concerned with. The question is how we display the list (given it's permissionless) so we can key in a constants or a daos file so that approved ones are on display for now.

Reply:
Ahhh I see yes it's that initial accounts.dao.all() call that renders the full list, and then when they select one we persist that in the state. And, I think we can do all that with react-query. I will add a method here to fetch all the DAOs so we can leverage that.